lib/zendesk_apps_support/validations/libraries.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module ZendeskAppsSupport
+  module Validations
+    module Libraries
+      GARDEN_CDN_URL = /<link\s.*href="(https?:\/\/\S*assets\.zendesk\.com\S*zendesk_garden\.css).*>/
+      LEGACY_SDK_URL = /<script\s.*src="(https?:\/\/\S*assets\.zendesk\.com\S*\/((\d(\.\d+){1,2})|latest)\/zaf_sdk\.js).*>/
```

Comment:
We should probably update the [docs](https://github.com/zendesk/zaf_docs/blob/16071338868b19975d37307c52d0654b24c06757/doc/v2/dev_guide/getting_started.md#L25) first 😅

Reply:
Yeah that's fine, and can do. I just added this while I was adding the Garden thing since it seemed I could collate them both underneath the `Validations::Libraries` banner